pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/statics/DBConstants.java

```diff
@@ -2,6 +2,8 @@
 
 public class DBConstants {
 
+    public static final String RGP = "RGP";
```

Comment:
How about moving the logic for "isRGP()" into DDPInstance?  A bit easier to find for re-use, and more general purpose than `isRgpUpload()`, which implies it's only suitable for upload-related checks.